.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "eslint.workingDirectories": [
```

Comment:
i'm not sure what this does, but in the past i've used a `[monorepo name].code-workspace` file for opening the monorepo in VSC. like this:
```ts
{
  "folders": [
    {
      "path": ".",
      "name": "[monorepo name]"
    },
    {
      "path": "apps/first-app",
      "name": "💻 first-app"
    },
    {
      "path": "apps/second-app",
      "name": "💻 second-app"
    },
    ...
    {
      "path": "packages/pkg1",
      "name": "📦 pkg1"
    },
    {
      "path": "packages/pkg2",
      "name": "📦 pkg2"
    },
    ...
  ],
  "settings": {
    "liveServer.settings.multiRootWorkspaceName": "[monorepo name]"
  }
}
```

Reply:
not sure if that serves a similar purpose or if it's something we should add in addition to this.

Reply:
hmm interesting